src/components/entity/map-popup.vue

```diff
@@ -10,17 +10,80 @@ including this file, may be copied, modified, propagated, or distributed
 except according to the terms contained in the LICENSE file.
 -->
 <template>
-  <div></div>
+  <map-popup v-show="uuid != null" ref="popup" :back="odata != null"
+    @hide="$emit('hide')" @back="$emit('back')">
+    <template #title>
+      <span v-tooltip.text>{{ entity.dataExists ? entity.label : '' }}</span>
+    </template>
+    <template #body>
+      <loading :state="entity.awaitingResponse"/>
+      <dl v-if="dataset.dataExists" v-show="entity.dataExists">
+        <div>
+          <dt>{{ $t('entity.entityId') }}</dt>
+          <dd>{{ props.uuid }}</dd>
+        </div>
+        <div>
+          <dt>{{ $t('header.createdAt') }}</dt>
+          <dd>
+            <date-time :iso="entity.dataExists ? entity.createdAt : null"/>
+          </dd>
+        </div>
+        <div v-for="property of dataset.properties" :key="property.name">
+          <dl-data :name="property.name"
+            :value="entity.dataExists ? entity[property.odataName] : null"/>
+        </div>
+      </dl>
+    </template>
+  </map-popup>
 </template>
 
 <script setup>
+import { inject, reactive, useTemplateRef, watch } from 'vue';
+
+import DateTime from '../date-time.vue';
+import DlData from '../dl-data.vue';
+import Loading from '../loading.vue';
+import MapPopup from '../map/popup.vue';
+
+import { apiPaths } from '../../util/request';
+import { useRequestData } from '../../request-data';
+
 defineOptions({
   name: 'EntityMapPopup'
 });
-defineProps({
+const props = defineProps({
   uuid: String,
   odata: Object,
   awaitingResponse: Boolean
 });
-defineEmits(['hide', 'back']);
+const emit = defineEmits(['hide', 'back']);
+
+const projectId = inject('projectId');
+const datasetName = inject('datasetName');
+
+const { dataset, createResource } = useRequestData();
+const entity = createResource('entity', () => ({
+  transformResponse: ({ data }) => reactive(data.value[0])
+}));
+
+const fetchData = () => {
+  const url = apiPaths.odataEntities(projectId, datasetName, {
+    $filter: `__id eq '${props.uuid}'`
+  });
```

Comment:
Let's fetch the entity from the REST endpoint rather than the OData endpoint:

- If we use `$filter` like this, then if the entity doesn't exist, there won't be a 404. Frontend would have to handle it.
- We already have a utility function to convert from OData to REST format. We can use that when the overlap popup passes through the OData that it has.
- Some modals that this popup will interact with expect the entity to be in REST format, e.g., `EntityUpdate`.